base/sysinfo.jl

```diff
@@ -188,6 +195,15 @@ function _cpu_summary(io::IO, cpu::AbstractVector{CPUinfo}, i, j)
     println(io)
 end
 
+"""
+    Sys.cpu_summary(io::IO=stdout, cpu::AbstractVector{CPUinfo} = cpu_info())
+
+Print a summary of information about each processor in `cpu` gathered from [`cpu_info](@ref). This is generally:
+  - The number (e.g. CPU 3 on a system with 10 total)
```

Comment:
I don't really understand the comment after the "e.g.".

Reply:
What I was trying to express is that if you have a node with 4 CPUs, it will print CPU 1 of 4, speed X, etc..

Reply:
How about this:

```suggestion
  - Its index in the `cpu` vector
```

Since:

```
julia> Sys.cpu_summary(stdout, Sys.cpu_info()[[1,1,1,1,2,2,2,3,4]])
Intel(R) Core(TM) i5 CPU       M 520  @ 2.40GHz:
       speed         user         nice          sys         idle          irq
#1  2400 MHz      13857 s          0 s       7446 s      57747 s          0 s
#2  2400 MHz      13857 s          0 s       7446 s      57747 s          0 s
#3  2400 MHz      13857 s          0 s       7446 s      57747 s          0 s
#4  2400 MHz      13857 s          0 s       7446 s      57747 s          0 s
#5  2400 MHz       6246 s          0 s       1995 s      70799 s          0 s
#6  2400 MHz       6246 s          0 s       1995 s      70799 s          0 s
#7  2400 MHz       6246 s          0 s       1995 s      70799 s          0 s
#8  2400 MHz      13639 s          0 s       5278 s      60124 s          0 s
#9  2400 MHz       6497 s          0 s       1923 s      70620 s          0 s
```

Also note that if you have more than 10 processors, it'll fold them all together and sum the columns.

```
julia> Sys.cpu_summary(stdout, Sys.cpu_info()[[1,1,1,1,2,2,2,2,3,4]])
Intel(R) Core(TM) i5 CPU       M 520  @ 2.40GHz:
          speed         user         nice          sys         idle          irq
#1-10  2400 MHz     100940 s          0 s      45149 s     648811 s          0 s
```